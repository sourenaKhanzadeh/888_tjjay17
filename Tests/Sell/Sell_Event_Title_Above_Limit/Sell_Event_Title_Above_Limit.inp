login
sell-standard
sell
Title here is over twenty five characters