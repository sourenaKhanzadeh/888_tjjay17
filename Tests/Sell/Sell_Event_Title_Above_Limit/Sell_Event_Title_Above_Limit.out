Welcome to the system
Enter session type
Enter operation
Enter the event title you would like to sell
Please re-enter an event that is between 1 and 25 characters long
Session terminated
